ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1137,7 +1218,12 @@
 
         # Set to false to disable interface validation by the `eos_validate_state` role.
         validate_state: <bool>
+
+        # This should not be combined with `ethernet_interfaces[].type = switched/routed`.
         switchport:
+
+          # Warning: This should not be combined with `ethernet_interfaces[].type = routed`.
+          enabled: <bool>
           port_security:
```

Comment:
Let's adjust the order of keys here. Port security is not used as often as the more general keys.
- enabled
- mode
- access
- trunk
- phone
etc.

Reply:
Same for port-channels.

Reply:
Fixed